src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileZipGeneratorHelperImpl.java

```diff
@@ -143,6 +146,26 @@ public String generateReadme(ViqMetaData viqMetaData, String fileLocation) {
         return readmeFile.getAbsolutePath();
     }
 
+    @Override
+    public Path generateViqFile(AudioFileInfo audioFileInfo, Path viqOutputFile) {
+
+        if (!audioFileInfo.isTrimmed()) {
```

Comment:
I don't think we need this bit tbh - what do you think @hemantasharma1129 ?

Reply:
the ticket requirement was to add the header just for trimmed files. Let's see if Hemanta confirms, if not I'll change this. Cheers

Reply:
Its only for the Audio we manipulated i.e. trimmed.

Reply:
Which is all the files currently :)